### PR TITLE
Add step to test persistent

### PR DESCRIPTION
**Pull request by duanwei33**
@qinpingli @chao007 
As discussed, I'd like add some steps to test volume "persistent", please help review.

[pass log](https://privatebin-it-iso.int.open.paas.redhat.com/?9a5edb3354024989#YiwLSkVLFK3vK9t5iaEPvl4fK5uT18DZkxzzC918pQw=)
There is one step "And I verify that the IAAS volume with id "<%= cb.volumeID %>" was deleted" need to be fixed in another thread.

### CODE REVIEW

File: features/storage/dynamic_provisioning.feature [hunk 1]
```diff
@@ -8,20 +8,42 @@ Feature: Dynamic provisioning
...
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | action=storage |
+    When I execute on the pod:
+      | ls | -l | /mnt/<cloud_provider>/testfile_1 |
+    Then the step should succeed
+    When I run the :delete client command with:
+      | object_type       | deployment |
+      | object_name_or_id | mydep      |
+    Then the step should succeed
```
Comment (liangxia) [inline features/storage/dynamic_provisioning.feature:45]: Given I ensure "mydep" deployment is deleted

Comment (duanwei33) [reply features/storage/dynamic_provisioning.feature]: Thanks, updated.

**Pull request merged**